templates/desktop/developers.html

```diff
@@ -501,6 +501,32 @@ <h3>Certified hardware</h3>
       </div>
     </div>
   </section>
+  <section class="p-strip--light">
+    <div class="row">
+      <div class="col-8">
+        <h2>What's new in Ubuntu {{ releases.lts.short_version }}</h2>
```

Comment:
This needs to say 22.10 but it says 22.04 now... is it gonna be updated automatically?

Reply:
yeah, this is changed in releases.yaml in [this PR](https://github.com/canonical/ubuntu.com/pull/12174/files) that's why this PR is mergeing into a feature branch `[22-10-release](https://github.com/canonical/ubuntu.com/tree/22-10-release)`

Reply:
I moved the yaml file to this PR, so please can you review